src/main/kotlin/se/uulm/snowballr/backend/service/EmailService.kt

```diff
@@ -0,0 +1,71 @@
+package se.uulm.snowballr.backend.service
+
+import io.github.oshai.kotlinlogging.KotlinLogging
+import org.simplejavamail.MailException
+import org.simplejavamail.api.mailer.Mailer
+import org.simplejavamail.email.EmailBuilder
+import org.simplejavamail.mailer.MailerBuilder
+import se.uulm.snowballr.backend.env.EnvReader
+
+private val logger = KotlinLogging.logger {}
+
+interface IEmailService {
+    /**
+     * Sends an email with the specified subject and HTML body to the given recipient.
+     *
+     * @param to The recipient's email address.
+     * @param subject The subject of the email.
+     * @param htmlBody The HTML content of the email.
+     *
+     * TODO: This method will be replaced by more specific methods like `sendInvitationEmail` in later issues
+     */
+    fun sendEmail(to: String, subject: String, htmlBody: String)
+}
+
+class EmailService(
+    private val envReader: EnvReader,
+) : IEmailService {
+    private val mailer: Mailer
+
+    init {
+        val env = envReader.env
+
+        logger.debug { "Initializing Mailer for email service" }
```

Comment:
This is an important step of the setup and should be logged not just for debugging.
```suggestion
        logger.info { "Initializing Mailer for email service" }
```